src/autoit_instance_symbols.cpp

```diff
@@ -10,25 +10,31 @@ DWORD autoit_path_size = 0;
 WCHAR autoit_path[MAX_PATH + 1] = {0};
 
 autoit_instance_symbols::autoit_instance_symbols(void) {
-	if (autoit_loaded) {
-		GetTempFileNameW(L".", L"ait", 0, this->library_path);
-		CopyFileW(autoit_path, this->library_path, false);
-		this->handle = LoadLibraryW(this->library_path);
-		this->is_library_clone = true;
-	} else {
-		this->handle = LoadLibraryW(AUTOIT_LIBRARY_NAME);
-		autoit_path_size = GetModuleFileNameW(this->handle, autoit_path, countof(autoit_path));
-		wcscpy_s(this->library_path, countof(this->library_path), autoit_path);
-		autoit_loaded = true;
-	}
-	void __stdcall (*init)(void) = (void __stdcall (*)(void))GetProcAddress(this->handle, "AU3_Init");
-	init();
-	#define X(type, name, params...) this->name = (type __stdcall (*)(params))(void*)GetProcAddress(this->handle, #name);
-	# include "autoit_functions.hpp"
-	#undef X
+  if (autoit_loaded) {
+    GetTempFileNameW(L".", L"ait", 0, this->library_path);
+    CopyFileW(autoit_path, this->library_path, false);
+    this->handle = LoadLibraryW(this->library_path);
+    this->is_library_clone = true;
+  } else {
+    this->handle = LoadLibraryW(AUTOIT_LIBRARY_NAME);
+    autoit_path_size =
+        GetModuleFileNameW(this->handle, autoit_path, countof(autoit_path));
+    wcscpy_s(this->library_path, countof(this->library_path), autoit_path);
+    autoit_loaded = true;
+  }
+  void __stdcall (*init)(void) =
+      (void __stdcall (*)(void))GetProcAddress(this->handle, "AU3_Init");
+  init();
+#define X(type, name, params...)                                               \
+  this->name =                                                                 \
+      (type __stdcall(*)(params))(void *)GetProcAddress(this->handle, #name);
+#include "autoit_functions.hpp"
+#undef X
+  this->AU3_AutoItSetOption(L"WinSearchChildren", 1);
```

Comment:
This is the only actual change?

Reply:
Yes but it has a global impact on our end: with this all functions which match windows (`WinWait`, `WinExists`, etc.) will also match nested windows.

Reply:
can it cause issues with other PMSs ?